prov/usnic/src/usnic_direct/vnic_rq.h

```diff
@@ -119,7 +119,7 @@ struct vnic_rq {
 #endif
 
 #ifdef CONFIG_NET_RX_BUSY_POLL
-	atomic_t bpoll_state;
+	atomic32_t bpoll_state;
```

Comment:
> ok, thank you. I didn't know about that due to lack of expertise in usnic.

No problem, it's pretty confusing that there are two different `atomic_t` types there.